pypeit/core/datacube.py

```diff
@@ -58,10 +58,13 @@ class DataCube(datamodel.DataContainer):
             Build from PYP_SPEC
 
     """
-    version = '1.0.1'
+    version = '1.0.2'
 
-    datamodel = {'flux': dict(otype=np.ndarray, atype=np.floating, descr='Flux array in units of counts/s/Ang or 10^-17 erg/s/cm^2/Ang'),
+    datamodel = {'flux': dict(otype=np.ndarray, atype=np.floating, descr='Flux array in units of counts/s/Ang/arcsec^2'
```

Comment:
I'm not following why this is either or?

Reply:
Okay, that is fine but then counts are not counts/s/ang unless you have divided by the exposure time (easy) and the delta_lambda (more complicated since in general it varies with pixel). So has that been done here (perhaps I missed it) or is the label of the units incorrect?

Reply:
Thanks for checking - the units label is correct here. The exposure time and delta wave is done around line 1200. The delta wave is done on a per pixel basis (as you mention, it's not constant for every pixel, and this is accounted for).